tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "include": ["src"],
   "exclude": ["**/*.spec.ts", "**/*.test.ts"],
   "compilerOptions": {
-    "module": "CommonJS",
+    "module": "ESNext",
```

Comment:
Which effects does this have? Can this change be breaking?

Reply:
This is required by `tsdx`. I actually just revert the commit of #97